assets/app/view/game/part/city.rb

```diff
@@ -228,6 +302,14 @@ def render_part
           h(:g, props, children)
         end
 
+        # look for a location for revenue based on edge and region_use
+        def find_revenue_location(edge)
+          revenue_location = REVENUE_LOCATIONS_BY_EDGE[edge].min_by do |cand|
```

Comment:
what's cand?

Reply:
i think you can just do this inline on line 335

revenue_location = REVENUE_LOCATIONS_BY_EDGE[edge].min_by { |loc| combined_cost(loc[:regions]) }
regions = revenue_location[...]
rotation = revenue_location[...]

Reply:
Yeah, this was originally much more complicated than it needed to be. I simplified it down and should have just skipped the added method.